wee_slack.py

```diff
@@ -2925,6 +2925,26 @@ def msg_command_cb(data, current_buffer, args):
             channel.send_message(message)
     return w.WEECHAT_RC_OK_EAT
 
+@slack_buffer_required
+def command_channels(data, current_buffer, args):
+    e = EVENTROUTER
+    team = e.weechat_controller.buffers[current_buffer].team
+
+    team.buffer_prnt("Channels:")
+    for channel in team.get_channel_map():
+        team.buffer_prnt(channel)
```

Comment:
I would still like these a bit indented, so you can see what they belong to and where they end. I.e. include four spaces before the channel names.